src/wallet/wallet.cpp

```diff
@@ -321,14 +321,12 @@ bool CWallet::LoadCScript(const CScript& redeemScript)
     }
 
     //fixme: (GULDEN) (FUT) (WATCHONLY)
-    bool ret = false;
     for (auto accountPair : mapAccounts)
     {
-        ret = accountPair.second->AddCScript(redeemScript);
-        if (ret == true)
+        if (accountPair.second->AddCScript(redeemScript))
             break;
```

Comment:
Did you mean to change this to "return true" perhaps? - otherwise it alters the program behaviour if/when AddCScript returns true.

Reply:
Hi Malcolm, thanks for your response :D 

I will do that, also I think I found another error with the return so this could should be left as is, I will reverse that now.